src/server/pbsd_init.c

```diff
@@ -1811,7 +1811,7 @@ pbsd_init_job(job *pjob, int type)
 		}
 
 		/* update entity limit sums for this job */
-		(void)account_entity_limit_usages(pjob, NULL, NULL, INCR);
+		(void)account_entity_limit_usages(pjob, NULL, NULL, INCR, ETLIM_ACC_ALL);
```

Comment:
Here we have to `INCR`all the 4 limit attributes. Hence passing `ETLIM_ACC_ALL`

Reply:
What happens if there are running jobs in the system. Would it affect queued limits?

Reply:
Yes this should be addressed. I'm thinking of handling this after I implement a PTL test case for restart scenario

Reply:
@Shrini-h how did you address this issue?

Reply:
I did not have to address this as from the server restart scenario in PTL, verifies that it is working properly